src/ngx_http_lua_util.h

```diff
@@ -260,6 +260,23 @@ void ngx_http_lua_set_sa_restart(ngx_log_t *log);
     }
 
 
+static ngx_inline size_t
+ngx_http_lua_safe_header_field_len(u_char *str, size_t len)
+{
+    size_t i;
+
+    for (i = 0; i < len; i++) {
+        if (*str == 13 || *str == 10) {
```

Comment:
I think it's more readable to write `\r` and `\n` instead. The performance is exactly the same.